### PR TITLE
changed everything associated with lolstats to porogram

### PR DESCRIPTION
**Pull request by joh0531**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# LOL Stats
+# Porogram
 Statistics interface for League of Legends
 ## Getting Started
 ```
-$ git clone git@github.com:lolstats/lolstats.git
+$ git clone https://github.com/Porogram/Porogram.git
```
Comment (edat44) [inline README.md]: Is there a reason this was changed from `ssh` to `https` in the readme?

**Comment by joh0531** [comment]
@edat44 would you be able to look into reconfiguring the ci to porogram instead of lolstats? I think circleci thinks its still associated with lolstats and is not working

File: README.md (hunk 1 above)
Comment (joh0531) [reply README.md]: nah should we keep as ssh?

Comment (edat44) [reply README.md]: I don't really care either way, I was just curious

Comment (joh0531) [reply README.md]: want me to merge rn or wait until the ci is fixed?

**Pull request merged**